src/test/java/com/hydratereminder/dictionary/HydrateWelcomeMessageDictionaryTest.java

```diff
@@ -1,18 +1,29 @@
 package com.hydratereminder.dictionary;
 
 import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
 
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNotNull;
+import java.security.SecureRandom;
 
+import static org.junit.Assert.*;
```

Comment:
Please do not add the wildcard imports.

https://stackoverflow.com/questions/147454/why-is-using-a-wild-card-with-a-java-import-statement-bad